site/content/internal/desktop-release-process.md

```diff
@@ -0,0 +1,117 @@
+---
+title: Desktop Release Process
```

Comment:
1. If you'd like, feel free to add steps from this doc to the existing Playbook we have https://community.mattermost.com/playbooks/playbooks/h3a39biacpnuim7ufmwiuuoxfo/outline.
2. I can also add a link to this page from the Handbook release process page https://handbook.mattermost.com/operations/research-and-development/product/release-process/desktop-release.

Reply:
@devinbinnie - friendly bump on Amy's feedback.

Reply:
Missed this as well 😅 

I can have a look at the playbook and make sure these line up as well :)
A link to this page would definitely be helpful.